packages/toolpad-app/src/components/AppEditor/PageEditor/QueryEditor.tsx

```diff
@@ -384,13 +410,26 @@ function QueryNodeEditorDialog<Q, P>({
                     />
                   }
                 />
-
-                <JsExpressionEditor
-                  globalScope={{}}
-                  value={input.attributes.transform?.value ?? '(data) => {\n  return data;\n}'}
-                  onChange={handleTransformFnChange}
-                  disabled={!input.attributes.transformEnabled?.value}
-                />
+                <Stack direction={'row'} spacing={2}>
```

Comment:
Wondering whether we shouldn't make this all just part of the `JsExpressionEditor`. Since it's very similar to what's being used in the binding editor?

Reply:
I think it's okay to keep them separate because that keeps it flexible for the user to configure them as needed in the area of the codebase where they are getting used